packages/webpack/src/plugin.js

```diff
@@ -284,6 +284,20 @@ class LavaMoatPlugin {
         const isContextModule = (m, moduleClass) =>
           moduleClass === 'ContextModule'
 
+        /**
+         * Identifies an asset that webpack includes in dist by default without
+         * setting any loaders explicitly.
+         *
+         * @param {import('webpack').Module} m
+         * @returns {m is import('webpack').NormalModule}
+         */
+        const isAmbientAsset = (m) =>
```

Comment:
same as i told @boneskull, i don't think `in` is the question we intend to ask here, and i think it's important we'd be explicit about the question we do intend to ask

Reply:
note that the `in` questions are there for typescript to shut up about me later using loaders. What I'm asserting that's relevant to security is m.type and m.loaders being an empty array. 
The rest is to stop typescript from complaining I use loaders and resource on a Module instance because there's no better type I can give it without lying and I know for a fact these fields are there.